app/src/main/java/ch/epfl/skysync/components/FlightDetail.kt

```diff
@@ -194,6 +214,40 @@ fun ConfirmFlightTimes(confirmedFlight: ConfirmedFlight, padding: Dp, cardColor:
   }
 }
 
+/**
+ * Composable function for displaying the takeoff and landing times for a finished flight.
+ *
+ * @param finishedFlight The finished flight object containing details to be displayed.
+ * @param padding The padding value to be applied around the content.
+ * @param cardColor The color to be used for the card background.
+ */
+@Composable
+fun FinishedFlightTimes(finishedFlight: FinishedFlight, padding: Dp, cardColor: Color) {
+  val times =
+      mapOf(
+          "Takeoff time" to finishedFlight.takeOffTime,
+          "Landing time" to finishedFlight.landingTime)
+
+  Card(colors = CardDefaults.cardColors(cardColor)) {
+    LargeTitle(title = "Operational times", padding = padding, color = Color.Black)
+    times.forEach { (label, time) ->
+      DisplaySingleMetric(
+          metric = label, value = DateUtility.localTimeToString(DateUtility.dateToLocalTime(time)))
+    }
+    DisplaySingleMetric("Flight Duration", longToTime(finishedFlight.flightTime))
```

Comment:
Why not add this in the `map` ?

Reply:
Did not think to do it like this but it's quite smart